index.html

```diff
@@ -33,6 +33,9 @@
   <!-- Search -->
 
   <!-- Fixed Side Nav -->
```

Comment:
The UI has been updated to work with columns. I would make sure to pull before you merge this as there will be conflicts